searx/engines/odysee.py

```diff
@@ -0,0 +1,92 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
```

Comment:
I added the [license tag](https://github.com/searxng/searxng/blob/master/LICENSE) .. the `# lint: pylint` instructs our CI to perform a pylint with a more rigorous assessment --> all new implementation must withstand these rigorous assessments.